src/monitoring_service/schema.sql

```diff
@@ -9,7 +9,8 @@ INSERT INTO blockchain DEFAULT VALUES;
 
 
 CREATE TABLE token_network (
-    address                 CHAR(42) PRIMARY KEY
+    address                 CHAR(42) PRIMARY KEY,
+    settle_timeout          INT
```

Comment:
Since this comes from a uint256, it could theoretically overflow SQLite's int type. Practically, it probably can't.

Reply:
So better use `HEX_INT`? 🤔

Reply:
It's a bit ugly when looking at the db tables, but that is what I chose to be on the safe side whenever uint256 were stored.

Reply:
Not sure if it's worth to change, but `settle_timeout` isn't variable per `TokenNetwork`, but constant for all TNs in a `TokenNetworkRegistry`. Maybe we could avoid storing it, and instead just recover it from TNR every startup, and avoid having to deal with the field at all?